libs/application/templates/social-insurance-administration/household-supplement/src/fields/Review/review-groups/Comment.tsx

```diff
@@ -0,0 +1,39 @@
+import { DataValue, ReviewGroup } from '@island.is/application/ui-components'
+import { GridColumn, GridRow } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { householdSupplementFormMessage } from '../../../lib/messages'
+import { ReviewGroupProps } from './props'
+import { getApplicationAnswers } from '../../../lib/householdSupplementUtils'
+
+export const Comment = ({
+  application,
+  editable,
+  goToScreen,
+}: ReviewGroupProps) => {
+  const { comment } = getApplicationAnswers(application.answers)
+  const { formatMessage } = useLocale()
+
+  return (
+    // eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
if you remove this line and the <> </>  and the { } you should be able to skip this eslint disable.

Something like
```
  return (
    comment && (
      <ReviewGroup
        isLast
        isEditable={editable}
        editAction={() => goToScreen?.('comment')}
      >
        <GridRow>
          <GridColumn span={['10/12', '10/12', '10/12', '10/12']}>
            <DataValue
              label={formatMessage(
                householdSupplementFormMessage.comment.commentSection,
              )}
              value={comment}
            />
          </GridColumn>
        </GridRow>
      </ReviewGroup>
    )
  )
```